datalad/distribution/publish.py

```diff
@@ -292,14 +353,19 @@ def __call__(
 
         if dataset and since:
```

Comment:
Forget my comments. Talkin' alternative facts ... believe me - a second ago, it did make sense. :-D